src/applications/personalization/dashboard-2/tests/e2e/benefits-of-interest.cypress.spec.js

```diff
@@ -32,8 +32,8 @@ describe('The My VA Dashboard', () => {
       user.data.attributes.vaProfile.facilities = [];
       cy.intercept('/v0/health_care_applications/enrollment_status', notInESR);
       cy.intercept('/v0/profile/ch33_bank_accounts', dd4eduNotEnrolled);
-      cy.login(user);
       mockFeatureToggles();
+      cy.login(user);
```

Comment:
`login()` needs to happen after mocking feature flags now? 🤔

Reply:
This was a test to see if it would fix a failure. I'll likely revert this back once I'm done with testing.